web_frontend/config/settings/common.py

```diff
@@ -54,6 +54,7 @@
     'osmaxx.version',
 
     'osmaxx.countries',
+    'osmaxx.conversion_api',
```

Comment:
Is this or will this be an app, or is it just a python module? Or did treating this as a normal module lead to any unexpected errors?